Gemfile

```diff
@@ -36,11 +36,12 @@ gem "redcarpet", "~> 3.6.0"
 
 # Use Active Storage variants [https://guides.rubyonrails.org/active_storage_overview.html#transforming-images]
 gem "image_processing", "~> 1.13.0"
+gem "ffi", "~> 1.15.5" # explicitly requiring 15.5 until this is resolved: https://github.com/ffi/ffi/issues/1105
```

Comment:
this is the only thing that actually fixed it. Maybe there will be another solution, but seems like our target platforms  aren't supported by the latest version of that gem so I don't think we can use it